docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "2.2"
+services:
+  database:
+    image: metaskills/mssql-server-linux-rails
+    ports:
+      - "1433:1433"
```

Comment:
Does this port need to be forwarded to the host? Ports should be automatically available to other containers here in the compose context

Reply:
It doesn't. Will remove that as well. Good spot.

Reply:
@aharpervc removed that as well.